### PR TITLE
security ootb deeplinks - [WEB-4345]

### PR DESCRIPTION
**Pull request by StefonSimmons**
### What does this PR do? What is the motivation?

adds `View Rule In Datadog` button and a view signals button that directs users to the product. 
- view signals button can read as:
    - `View Signals`  for Cloud SIEM, Application Security and CSM Threat rules
    - `View Identity Risks` for CSM Identity Risks rules
    - `View Misconfigurations` for CSM Misconfiguration rules
  
realigns  the`.rule-specification` section's elements to include the new buttons

jira: https://datadoghq.atlassian.net/browse/WEB-4345

preview: https://docs-staging.datadoghq.com/stefon.simmons/sec-ootb-deeplinks/security/default_rules/
- Filter for rules by category. Ensure the deeplink buttons on the rule pg display the correct text according to the rule category,  and direct you to **Datadog**.

rule pg preview examples:
- https://docs-staging.datadoghq.com/stefon.simmons/sec-ootb-deeplinks/security/default_rules/auditd_modification/#all
- https://docs-staging.datadoghq.com/stefon.simmons/sec-ootb-deeplinks/security/default_rules/tool-scanner-detected/#all
- https://docs-staging.datadoghq.com/stefon.simmons/sec-ootb-deeplinks/security/default_rules/aws-cloudtrail-kms-deleted/#all
- https://docs-staging.datadoghq.com/stefon.simmons/sec-ootb-deeplinks/security/default_rules/1password-activity-from-tor-ip/#all
- https://docs-staging.datadoghq.com/stefon.simmons/sec-ootb-deeplinks/security/default_rules/blast-radius-group/
- https://docs-staging.datadoghq.com/stefon.simmons/sec-ootb-deeplinks/security/default_rules/aws-cloudfront-integrated-with-waf/
- https://docs-staging.datadoghq.com/stefon.simmons/sec-ootb-deeplinks/security/default_rules/sec-issue-aws-cryptominer_args/

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: layouts/security_rules/single.html [hunk 1]
```diff
@@ -22,38 +22,65 @@ <h1 id="pagetitle">{{ .Title }}</h1>
...
+				{{ end }}
+			</div>
+		</div>
+		<div class="col-12 col-md-9 col-lg-10 mt-1">
+			<div class="row">
+				{{ $rule_category := (index .Params.rule_category 0)}}
+				{{ $view_findings_text := "View Signals"}}
+
+				{{ if eq $rule_category "CSM Identity Risks"}}
+				{{ $view_findings_text = "View Identity Risks"}}
```
Comment (bgdeutsch) [inline layouts/security_rules/single.html:65]: hey @StefonSimmons would you mind fixing the indentation here?

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```
Comment (brett0000FF) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```

**Review by brett0000FF — CHANGES_REQUESTED** [review]
Wow, this is really cool! Thanks for all your work on this. I just left one comment about switching `cache_enabled` back to `true`.

Otherwise, the new functionality worked great for me from the preview site.

**Review by bgdeutsch — APPROVED** [review]
LGTM 👍

**Review by brett0000FF — APPROVED** [review]

**Pull request merged**